cumulusci/tasks/bulkdata/step.py

```diff
@@ -429,7 +429,8 @@ def load_records(self, records):
         def _convert(rec):
             result = dict(zip(self.fields, rec))
             for boolean_field in self.boolean_fields:
-                result[boolean_field] = bool(result[boolean_field])
+                # This is clumsy but required since Booleans are stored as text.
+                result[boolean_field] = result[boolean_field].lower() == "true"
```

Comment:
Just to check, are we sure result[boolean_field] is always a string and not None?

Reply:
It can't be `null`/`None` unless the user manually edited the SQL. Booleans can be `null` in Apex but are always stored and returned to SOQL as `False`.

Reply:
I suppose if we're loading from a Postgres dataset that was not generated from a Salesforce extract it could be `None`. I'll add a guard.